### PR TITLE
gridfs implementation

### PR DESCRIPTION
**Pull request by stephan271**

### CODE REVIEW

File: scilog/src/app/core/config/server-settings.service.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ export class ServerSettingsService {
 
   constructor() { }
 
-  public serverType: string = "production";
+  public serverType: string = "kwHome";
```
Comment (wakonig) [inline scilog/src/app/core/config/server-settings.service.ts:10]: This needs to be reverted...

Comment (stephan271) [reply scilog/src/app/core/config/server-settings.service.ts:10]: Good point

**Comment by wakonig** [comment]
You've mentioned today that it was already working. Did you also test the FE or just the BE?

**Review by wakonig — APPROVED** [review]
looks good to me! Thanks!

**Comment by stephan271** [comment]
I tested the backend by using the frontend, which worked fine ;-)

**Pull request merged**